src/js/utils/html-utils.js

```diff
@@ -129,7 +111,7 @@
                         },
                         data
                     );
-                    return HTML(rawTemplate(augmentedData));
+                    return rawTemplate(augmentedData);
```

Comment:
@andy-armstrong Your happier with this?  I thought someone needed to use `toString()` and the jQuery upgrade would improve this and allow for backward compatibility?  If you return a String here, you must always return a String here, and we wouldn't be able to improve it unless we added a new method like `templateAsHtmlSnippet`, which gets long enough that we may always have to require using the wrapper.

Would you mind determine if the jQuery upgrade really does influence this before we commit one way or another?